rainbow-cake-core/src/main/java/co/zsmb/rainbowcake/internal/logging/LogLevel.kt

```diff
@@ -0,0 +1,13 @@
+package co.zsmb.rainbowcake.internal.logging
+
+import android.util.Log
+
+public enum class LogLevel(public val priority: Int = Log.DEBUG) {
+    VERBOSE(Log.VERBOSE),
+    DEBUG(Log.DEBUG),
+    INFO(Log.INFO),
+    WARN(Log.WARN),
+    ERROR(Log.ERROR),
+    ASSERT(Log.ASSERT),
+    OFF(Int.MAX_VALUE);
+}
```

Comment:
We could use directly Log priority but it's more reliable to have this wrapper. But it's ok to use Int directly for all log levels (as `Timber` does it) if you want.